### ISSUE TITLE
Can't run without having sudo installed on the system

### ISSUE THREAD

**Issue by Andy3153** [hinto-janai/gupax#19]
Title says it all. If I only have `doas` on my system, I cannot start XMRig from the GUI.

Either don't depend on sudo this much, or allow running the entire GUI as root (not that recommended).

**Comment by hinto-janai** [comment]
Lots of software is broken if `sudo` isn't installed.

Since >99% of installations have `sudo`, the responsibility may be on you to make sure `doas` interacts with the rest of your system nicely.

**Comment by Andy3153** [comment]
Yeah, you're right that it's mostly on me to keep things working if I stray away from the standard, but I also find it a little weird that a GUI app requires sudo. Why not go for PolKit like all the other GUI programs (ex.: Etcher, GParted etc.)?

**Comment by hinto-janai** [comment]
See https://github.com/hinto-janai/gupax/issues/17.

**Comment by hinto-janai** [comment]
Unix as in macOS and Linux, Gupax doesn't have BSD binaries.

`sudo` works on both, there's no need to write platform-specific code.
